operator/pkg/manifest/manifest.go

```diff
@@ -0,0 +1,54 @@
+package manifest
+
+import (
+	"fmt"
+	"os"
+
+	operatorv1alpha1 "github.com/kyma-project/istio/operator/api/v1alpha1"
+	istioOperator "istio.io/istio/operator/pkg/apis/istio/v1alpha1"
+	"sigs.k8s.io/yaml"
+)
+
+var (
+	defaultIstioOperatorFile = "default-istio-operator-k3d.yaml"
```

Comment:
We could probably use https://github.com/kyma-project/cli/blob/16c49248da0432e7786757bac132d9741ee2a29e/internal/clusterinfo/clusterinfo.go#L43 for checking on which cluster we are
```suggestion
	defaultIstioOperatorFile = "default-istio-operator-k3d.yaml"
```

Reply:
I have just added a second manifest for the future, so we won't need to look at what we are missing.
This is currently out of the scope of this task.
That's a good idea, we will need to extend our configuration, so I think we should check it and apply different CNI config if needed.